docs/cmd/kn_source_api-server_create.md

```diff
@@ -0,0 +1,44 @@
+## kn source api-server create
+
+Create a Kubernetes Api Server event source
+
+### Synopsis
+
+Create a Kubernetes Api Server event source
+
+```
+kn source api-server create NAME --resources RESOURCE --accountname ACCOUNTNAME --sink SINK --mode MODE [flags]
+```
+
+### Examples
+
+```
+
+  # Create a Kubernetes Api Server event source 'k8sevents' which consumes Kubernetes events and send them to service 'mysvc' as a cloudevent
+  kn source api-server create k8sevents --resources Event --accountname myaccountname --sink svc:mysvc
+```
+
+### Options
+
+```
+      --accountName string   Name of the ServiceAccount to use to run this source
```

Comment:
Inconsistent use of camel case between `accountName` and `apiversion`.

Unless there is a strong reason, I'd suggest to be consistent and use camel case in all cases.

Reply:
done